src/.gitattributes

```diff
@@ -0,0 +1,3 @@
+
+# see https://github.com/github/linguist#generated-code
```

Comment:
This isn't necessary for scss generation is it? If it's not, can we raise a separate issue and PR to consider adding this to the repo?

Reply:
not directly nessesaary.
its used by to tell generated/vendored/... files apart from actual code.
sure, can make an extra PR for that.

Reply:
thought about it.
i feel like it should be part of the initial switch to SCSS right away. having it an extra PR feels not right, @gnikonorov

Reply:
This will just end up ignoring the generated files in the language breakdown on github, right?

If that's what this does I agree I was wrong and we should include it @jkowalleck. Can you confirm that's what this will do?

Reply:
thats what it does.